phpstan.neon

```diff
@@ -0,0 +1,13 @@
+includes:
+  - %currentWorkingDirectory%/vendor/prestashop/php-dev-tools/phpstan/ps-module-extension.neon
+    
+parameters:
+  paths:
+    - %currentWorkingDirectory%
+  reportUnmatchedIgnoredErrors: false
```

Comment:
This doesn't appear relevant as we do not have any ignore rules

Reply:
removed